lib/mixlib/install/generator/powershell/scripts/get_project_metadata.ps1.erb

```diff
@@ -36,9 +36,18 @@ function Get-ProjectMetadata {
     # Release channel to install from
     [validateset('current', 'stable')]
     [string]
-    $channel = 'stable'
+    $channel = 'stable',
+    # The following legacy switches are just aliases for the current channel
+    [switch]
+    $prerelease,
+    [switch]
+    $nightlies
```

Comment:
are we allowed to use this word? I feel like @schisamo set up some sort of punishment for doing such.
If we never supported these for the powershell script, we can probably dump them.